config/public/settings.py

```diff
@@ -55,6 +55,7 @@ class ExternalConfig(RemoteConfig):
     RELEASE_NOTES_URL = "https://forum.depmap.org/c/announcements/15"
     FORUM_URL = "https://forum.depmap.org/"
     FORUM_RESOURCES_CATEGORY = "resources-prototype"
+    FORUM_RESOURCES_DEFAULT_TOPIC_ID = 3396
```

Comment:
This maps to [this topic](https://forum.depmap.org/t/welcome/3396) . I don't believe it's an issue to show the identifier here even though this public forum topic is currently not available for public view yet.

Reply:
Yeah, I don't see a problem